### PR TITLE
Ensure FI map is rendered correctly when there is just one goal

### PR DESCRIPTION
**Pull request by moshthepitt**
It turns out that there was a bug that prevented points and structures being loaded on the map if there was only one goal.

This PR fixes that bug and adds tests to prevent its regression.

Fixes: https://github.com/onaio/reveal-frontend/issues/1397

### CODE REVIEW

File: src/containers/pages/FocusInvestigation/map/active/index.tsx [hunk 1]
```diff
@@ -539,7 +539,7 @@ const mapStateToProps = (state: Partial<Store>, ownProps: any): MapStateToProps
     });
   }
 
-  if (plan && jurisdiction && goals && goals.length > 1) {
+  if (plan && jurisdiction && goals && goals.length > 0) {
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/map/active/index.tsx:542]: This is the core bugfix.

File: src/containers/pages/FocusInvestigation/map/active/tests/__snapshots__/index.test.tsx.snap [hunk 2]
```diff
@@ -209,6 +209,17 @@ exports[`containers/pages/FocusInvestigation/activeMap renders SingleActiveFimap
 </Breadcrumb>
 `;
 
+exports[`containers/pages/FocusInvestigation/activeMap renders SingleActiveFimap correctly & changes page title: GisidaLite layers 1`] = `
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/map/active/tests/__snapshots__/index.test.tsx.snap:212]: We are codifying the layers rendered by the map.

File: src/containers/pages/FocusInvestigation/map/active/tests/__snapshots__/index.test.tsx.snap [hunk 3]
```diff
@@ -347,6 +358,13 @@ exports[`containers/pages/FocusInvestigation/activeMap renders correct detail vi
 </div>
 `;
 
+exports[`containers/pages/FocusInvestigation/activeMap works correctly with Redux store when there is just one goal: GisidaLite layers 1`] = `
+Array [
+  "main-plan-layer-313056f2-2a21-454e-9f88-63207acb00d4",
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/map/active/tests/__snapshots__/index.test.tsx.snap:363]: This test proves that the `RACD_register_families-point` layer is now passed on to GisidaLite.

**Review by p-netm — APPROVED** [review]

**Review by kahummer — APPROVED** [review]

**Comment by kahummer** [comment]
LGTM :+1:

**Pull request merged**